socialdistribution/profiles/models.py

```diff
@@ -75,7 +75,7 @@ class Author(AbstractBaseUser, PermissionsMixin):
     def url(self):
         # In the future use url reverse
         # reverse('author', args=[str(id)])
-        return("%s/author/%s" % (self.host, self.id))
+        return("%sauthor/%s" % (self.host, self.id))
```

Comment:
I'd suggest something like this:
```
if self.host.strip()[-1] == "/":
            return("%sauthor/%s" % (self.host, self.id))
        return("%s/author/%s" % (self.host, self.id))
```

Reply:
Fair. I made the assumption that the host would always end in a  slash. You're suggestion is better. I'll fix it.